client/js/socket-events/init.js

```diff
@@ -59,17 +59,17 @@ function openCorrectChannel(clientActive, serverActive) {
 
 	// Open last active channel
 	if (clientActive > 0) {
-		target = sidebar.find("[data-id='" + clientActive + "']");
+		target = sidebar.find(`.chan[data-id="${clientActive}"]`);
 	}
 
 	// Open window provided in location.hash
 	if (target.length === 0 && window.location.hash) {
-		target = $("#footer, #sidebar").find("[data-target='" + escape(window.location.hash) + "']");
+		target = $("#footer, #sidebar").find(`.chan[data-target="${escape(window.location.hash)}"]`);
 	}
 
 	// Open last active channel according to the server
 	if (serverActive > 0 && target.length === 0) {
-		target = sidebar.find("[data-id='" + serverActive + "']");
+		target = sidebar.find(`.chan[data-id="${serverActive}"]`);
```

Comment:
So, I could hit this block (by loading `http://localhost:9000/` which then adds the `#chan-X` fragment) and the previous one (by loading `http://localhost:9000/#chan-X`), but how do you reach the first one (`// Open last active channel`) and the last one (`// Open first available channel`)?
I couldn't find a way.